driver/native/native_test.go

```diff
@@ -119,3 +133,35 @@ func TestNativeDriverNativeParse_Malformed(t *testing.T) {
 	require.NotNil(err)
 	require.True(derrors.ErrDriverFailure.Is(err))
 }
+
+func TestNativeDriverParse_Timeout(t *testing.T) {
+	require := require.New(t)
+
+	// This test check if the Go driver server can recover from response read timeout.
+	//
+	// For this, we start a mock that will sleep 5 sec before answering any requests.
```

Comment:
We can probably use shorter times—5 sec. is a long time to make a test hang. I'd suggest like 1 sec. and 250ms.

Reply:
CI is sometimes extremely slow. This may lead to flaky behavior. But we can try.

Reply:
I noticed Travis is very slow to start up, but once it's running it doesn't seem too bad. But if you think it'll cause problems we don't need to change it, it just seems like a long hang.